tests/Functional/Schema/MySQLSchemaManagerTest.php

```diff
@@ -547,9 +546,6 @@ public function testListTableColumnsThrowsDatabaseRequired(): void
         $schemaManager = $connection->createSchemaManager();
 
         $this->expectException(DatabaseRequired::class);
-        $this->expectExceptionMessage(
-            'A database is required for the method: ' . AbstractSchemaManager::class . '::listTableColumns'
```

Comment:
This assertion now fails since the exception is thrown from within `doListTableColumns()`. The original message will be restored as a result of removing the BC layer but the exact exception message is irrelevant to the test.